app/lint.xml

```diff
@@ -13,5 +13,7 @@
         <ignore path="**/ic_person_24dp.png"/>
         <ignore path="**/ic_search_24dp.png"/>
     </issue>
-
+    <issue id="UnusedNamespace" severity="error"/>
```

Comment:
yes, this is the [list](http://tools.android.com/tips/lint-checks) of checks android lint already runs when we execute ./gradlew lint  or ./gradlew lintDebug
In lint.xml we could change the severity of these checks, hence changed the severity of  SdCardPath, NewerVersionAvailable and Typos issues from warnings to errors. Please correct me if I am wrong :)